src/style.css

```diff
@@ -0,0 +1,117 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
Add box-sizing: border-box for all elements (it should fix card width) and try to pass tests